Terminal.Gui/Views/GraphView/Annotations.cs

```diff
@@ -122,7 +122,19 @@ public class LegendAnnotation : IAnnotation {
 		/// <see cref="Bounds"/> and so reduces the width/height
 		/// available for text by 2
 		/// </summary>
-		public bool Border { get; set; } = true;
+		public bool Border {
+			get => _border;
+			set {
+				_border = value;
+				if (_border) {
+					_legend.BorderStyle = LineStyle.Single;
```

Comment:
It was my intent that simply changing BorderStyle to something other than None would automatically set the thickness to 1. However, this may be suprising behavior (I'm a big fan of following the "principle of least suprise"). Thoughts?

Reply:
I agree if BorderStyle is other than None then at least set the thickness to 1, if it's 0 and also agree if is None set the thickness to 0. If None is wanted without thickness the the user must set it to 0.

Reply:
Right. Deleted.